torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -1216,6 +1286,22 @@ RegisterOperators reg2({
         "aten::hash.generic(t value) -> int",
         hashValue,
         aliasAnalysisFromSchema()),
+
+#define DEFINE_COMPLEX_MIXED_OP(type_a, type_b)                      \
```

Comment:
Thanks for fixing the clang-tidy issues, there seems to be two more warnings left, could you take a look? Thanks.

Reply:
Yeah, this one suggests to define a 'constexpr' template function instead of a macro. Let me see if this can be disabled.